shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +146,146 @@ shared_library("create_flutter_framework_dylib") {
   ]
 }
 
+ocmock_path = "../../../../../third_party/ocmock/Source"
+
+# TODO: Clone the OCMock repository so we can add a BUILD.gn to it.
+static_library("ocmock") {
+  configs -= [ "//build/config/compiler:chromium_code" ]
+  cflags = [
+    "-fvisibility=default",
+    "-mios-simulator-version-min=13.0",
```

Comment:
Please move this variable to `//build/config/ios/ios_sdk.gni` and add this flag to sysroot flags in `//build/toolchain/mac/BUILD.gn`. See where we add `-miphoneos-version-min=`, this can go right next to that on simulator targets. Adding sysroot flags on a per target basis quickly gets out of hand.

Reply:
I pulled out the variable, added a todo issue to move it into build.  Since this PR already has many moving pieces I want to do it in a separate PR instead of getting another repo involved.